src/Controller/DefaultController.php

```diff
@@ -22,9 +23,18 @@ class DefaultController extends AbstractController
     /**
      * @Route("/", name="index", methods={"GET"})
      */
-    public function index(): Response
+    public function index(PostRepository $postRepository): Response
     {
-        return $this->render('views/index.html.twig');
+        $postRepository = $this->getDoctrine()->getRepository(Post::class)->findBy([
```

Comment:
Sorry I know you've changed this because of my previous comment, but you are now handling Posts and not the repository, the variable name should be `$posts`.  
Also (line 32), `findBy()` will always return an array, you should therefore test for `$posts`'length, and not wether it exists or not. **However**, we do not want to raise an exception in this case but handle the empty state in the frontend.

Reply:
Hello again! I fixed this issue too (not committed yet).